### PR TITLE
Revert Unstable on Expired Certs

### PR DESCRIPTION
**Pull request by cubastanley**
We now have a feature in the build to renew expired certs when the product is being built which makes this stage redundant and will only cause issues with PR testing - thus I am reverting it

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -30,10 +29,7 @@ pipeline {
                     echo "Domain name is ${DOMAIN_NAME}"
 
                     echo '*#*#*#*#*#*#*#*#*#*#*#*#  Certificates Expiring Soon  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
-                    rc = sh(returnStatus: true, script: "keytool -list -v -keystore \'${pwd()}/nucleus/admin/template/src/main/resources/config/cacerts.jks\' -storepass \'changeit\' | grep -E \"Valid from:.*`date +'%b' --date='+1 month'`.*`date +'%Y'`|`date +'%b'`.*`date +'%Y'`\"")
-                    if(rc != 1) {
-                        currentBuild.result = 'UNSTABLE'
-                    }
+                    sh """keytool -list -v -keystore '${pwd()}/nucleus/admin/template/src/main/resources/config/cacerts.jks' -storepass 'changeit' | grep -E 'Valid from:.*`date +'%b' --date='+1 month'`.*`date +'%Y'`|`date +'%b'`.*`date +'%Y'`"""
```
Comment (AlanRoth) [inline Jenkinsfile:32]: Can be removed also? I don't think anyone pays attention to this

**Review by AlanRoth — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (AlanRoth) [reply Jenkinsfile:32]: I even forgot about it

Comment (cubastanley) [reply Jenkinsfile:32]: Could but I'm just reverting the commit here, not doing any manual intervention

**Comment by cubastanley** [comment]
Jenkins test please

File: Jenkinsfile (hunk 1 above)
Comment (AlanRoth) [reply Jenkinsfile:32]: Keep the scripts clean, might as well

Comment (cubastanley) [reply Jenkinsfile:32]: Looks like I'll have to, causes jenkins scripts to fail

**Comment by cubastanley** [comment]
Jenkins test please

**Review by AlanRoth — APPROVED** [review]

**Comment by cubastanley** [comment]
Jenkins test please

**Pull request merged**